examples/js/loaders/ColladaLoader.js

```diff
@@ -969,11 +969,11 @@ THREE.ColladaLoader.prototype = {
 
 			if ( data.bindShapeMatrix ) {
 
-				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose(); 
+				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose();
 
 			} else {
 
-				build.bindMatrix = new THREE.Matrix4().transpose(); 
+				build.bindMatrix = new THREE.Matrix4().transpose();
```

Comment:
Granted this is just cleanup, but for clarity, I think it should be
```javascript
build.bindMatrix = new THREE.Matrix4().identity();
```
even though the matrix is initialized to the identity by default.